### PR TITLE
Update encounter_rewards.csv

### PR DESCRIPTION
**Pull request by MitchPleaseAllWorld**

### CODE REVIEW

File: content/data/players.csv [hunk 1]
```diff
@@ -1,81 +1,86 @@
 name,shortName,levelAvailable,startingRank,maxRank,type,team,division,conference,position1,position2,ballHandlingBase,perimeterShootingBase,midRangeShootingBase,dunkPowerBase,defenseBase,blockingBase,stealingBase,strengthBase,speedBase,staminaBase,ballHandlingMax,perimeterShootingMax,midRangeShootingMax,dunkPowerMax,defenseMax,blockingMax,stealingMax,strengthMax,speedMax,staminaMax,beatToReachRank2,beatToReachRank3,beatToReachRank4,beatToReachRank5,beatToReachRank6
 Darius Bazley,D. Bazley,1,1,4,Starter,PHX,Pacific,West,PF,,13,14,15,15,12,14,9,13,12,16,67,69,77,75,58,72,47,63,59,80,Markelle Fultz,Andre Drummond,De'Aaron Fox,,
-Bojan Bogdanovic,B. Bogdanovic,1,1,4,Starter,DET,Central,East,SF,PF,14,17,14,14,12,8,10,11,13,16,71,87,70,72,58,42,48,55,64,80,Larry Nance Jr.,Dillon Brooks,Lonzo Ball,,
```
Comment (unkdrew) [inline content/data/players.csv]: Should be `Jr.`, not `Jr`, since it’s an abbreviation.

File: content/data/players.csv [hunk 2]
```diff
@@ -1,81 +1,86 @@
...
-Dillon Brooks,D. Brooks,1,1,4,Starter,MEM,Southwest,West,SF,SG,14,14,15,12,14,9,12,12,13,16,70,72,77,61,70,46,58,58,64,80,TJ McConnell,Alec Burks,Pascal Siakam,,
+Bojan Bogdanovic,B. Bogdanovic,1,1,4,Starter,DET,Central,East,SF,PF,14,17,14,14,12,8,10,11,13,16,71,87,70,72,58,42,48,55,64,80,Larry Nance Jr,Dillon Brooks,Lonzo Ball,,
+Dillon Brooks,D. Brooks,1,1,4,Starter,MEM,SouthWest,West,SF,SG,14,14,15,12,14,9,12,12,13,16,70,72,77,61,70,46,58,58,64,80,TJ McConnell,Alec Burks,Pascal Siakam,,
```
Comment (unkdrew) [inline content/data/players.csv]: Should be `Southeast`, not `SouthEast`. Same with Southwest.

File: content/data/players.csv [hunk 3]
```diff
@@ -1,81 +1,86 @@
...
 Klay Thompson,K. Thompson,3,2,5,All-Star,GSW,Pacific,West,SG,SF,23.8,25.4,23.6,17.5,24,18.1,20.6,19.6,20.8,27,80,85,79,66,81,65,70,69,71,90,,Andre Drummond,Rudy Gobert,Malcolm Brogdon,
 Nikola Vucevic,N. Vucevic,3,2,5,All-Star,CHI,Central,East,C,,16.1,22,25.7,20.9,23.6,27,19.1,27,18.9,27,57,74,87,72,79,90,66,90,64,90,,Darius Bazley,?,?,
-Bam Adebayo,B. Adebayo,4,2,5,All-Star,MIA,Southeast,East,C,,17.5,19.3,25.1,23.6,25.5,26.7,20.9,27.3,20.8,27,61,67,83,79,86,88,72,92,71,90,,Buddy Hield,Nikola Vucevic,?,
+Bam Adebayo,B. Adebayo,4,2,5,All-Star,MIA,SouthEast,East,C,,17.5,19.3,25.1,23.6,25.5,26.7,20.9,27.3,20.8,27,61,67,83,79,86,88,72,92,71,90,,Buddy Hield,Nikola Vucevic,?,
 Jaylen Brown,J. Brown,4,2,5,All-Star,BOS,Atlantic,East,SG,SF,25.3,22.5,25.4,27.1,25.4,22.5,23.6,25.4,25.1,27,84,77,85,91,85,77,79,85,83,90,,Devonte Graham,Dejounte Murray,Kristaps Porzingis,
-CJ McCollum,C. McCollum,4,2,5,All-Star,NOP,Southwest,West,SG,PG,26.9,23.8,29.9,20.8,17.7,19.1,18.7,18.9,23.8,27,89,80,98,71,62,66,63,64,80,90,,Kevin Love,?,?,
+CJ Mccollum,C. McCollum,4,2,5,All-Star,NOP,SouthWest,West,SG,PG,26.9,23.8,29.9,20.8,17.7,19.1,18.7,18.9,23.8,27,89,80,98,71,62,66,63,64,80,90,,Kevin Love,?,?,
```
Comment (unkdrew) [inline content/data/players.csv]: Should be McCollum

**Review by unkdrew — CHANGES_REQUESTED** [review]

File: content/data/players.csv (hunk 3 above)
Comment (MitchPleaseAllWorld) [reply content/data/players.csv]: strange, I thought I changed all those things. I'll fix it again

Comment (unkdrew) [reply content/data/players.csv]: Just changed all NorthWest to Northwest, seems to be a miss :)

**Review by unkdrew — APPROVED** [review]

**Pull request merged**